### ISSUE TITLE
SmeshingService doesn't return supported GPU providers

### ISSUE THREAD

**Issue by avive** [spacemeshos/go-spacemesh#2660]
1. Build node binary from develop branch and copy it to a folder.
1. Copy the release artifacts of gpu-post from the node's build folder (downloaded on make) - note that this simulates what the Smapp installer is going to do on install on users' systems.
1. Get the gpu test app from the gpu-post latest release: https://github.com/spacemeshos/gpu-post/releases
1. Set execute permissions on the dynamic libs and the gpu-post test app.
1. Follow instructions in gpu-post repo regarding what you need to do to run the app and use the same dll(s) as go-spacemesh node. https://github.com/spacemeshos/gpu-post#running-the-test-app
1. Start the node (via smapp or stand-alone) with the current devent config file.
1. List GPU providers via the test app and via the node api:

```
grpcurl -d '{}' -plaintext localhost:9092 spacemesh.v1.SmesherService.PostSetupComputeProviders
```

```
 ./gpu-setup-test -l
```

Result:
Node only returns the CPU provider:
```
{
  "providers": [
    {
      "id": 1,
      "model": "CPU",
      "computeApi": "COMPUTE_API_CLASS_CPU"
    }
  ]
}
```

The gpu-post lib test app correctly returns system supported gpu providers. e.g.:

```
Available POST compute providers:
  1: [VULKAN] Apple M1
  2: [CPU] CPU
```

Expected:
- Node API to return all supported gpu providers on the system.

Thanks @IlyaVi for reporting this.

ENV:
- Rerpro on macOS/X86-64 adn on macOS/ARM/M1

**Comment by avive** [comment]
@moshababo

**Comment by avive** [comment]
I think this is fixed in v0.2.0-rc4 - let's leave it open until we can verify it by more clients but no deeper investigation is needed at this point.

**Comment by avive** [comment]
Fixed in latest devnet branch.
